packages/suite/src/components/suite/modals/AddAccount/index.tsx

```diff
@@ -149,6 +149,7 @@ export const AddAccount = ({ device, onCancel, symbol, noRedirect }: Props) => {
         <StyledModal
             isCancelable
             onCancel={onCancel}
+            onBackClick={selectedNetwork ? () => selectNetwork(undefined) : undefined}
```

Comment:
check this https://github.com/trezor/trezor-suite/blob/73452dbcc08c18240b5ba8d7eea60f2241d085cd/packages/suite/src/components/suite/modals/AddAccount/index.tsx#L57-L59

in coinmarket exchange there is a need to pin network and do not allow to change it

Reply:
8d0ff73572f1dde6c478bfbc72d1e1d850a331cb